alephium/caip2.md

```diff
@@ -0,0 +1,61 @@
+---
+namespace-identifier: alephium-caip2
+title: Alephium Blockchain ID Specification
+author: Hongchao Liu (@h0ngcha0)
+discussions-to: https://github.com/ChainAgnostic/namespaces/pull/117
+status: Draft
+type: Standard
+created: 2024-06-19
+updated: 2024-06-19
+---
+
+## Abstract
+
+In CAIP-2 a general blockchain identification scheme is defined.
+This is the implementation of CAIP-2 for Alephium network.
+
+## Rationale
+
+Alephium is a sharded blockchain that organizes addresses and states
+into distinct groups. The blockchain id includes both the network id
+and the group information.
+
+The Network id can be one of three values: `mainnet`, `testnet` and
+`devnet`.
+
+Currently, Alephium has 4 groups. The group component of the
+blockchain id can take one of five values: `0`, `1`, `2` and `3` which
+correspond to groups `0` through `3`, and `-1` which represents any
+group.
```

Comment:
I think what's missing here is an explanation of what a group _does_, and in what context "any group" would be meaningful.  for example: is there a 1:1 relationship between blockchains and groups? Does each group contain 4 blockchains, and each blockchain belong only to one group? 

It might also be worth mentioning _somewhere_, maybe in CAIP-19 or CAIP-10 or both, that smart-contracts are group-specific and that most wallets control an address in each group for this purpose, as [described here](https://docs.alephium.org/frequently-asked-questions/#why-can-the-dapp-only-connect-to-one-of-my-addresses) -- that's probably something crucial that multi-chain dapp or multichain-wallet developers will need a heads up about!

Reply:
Thanks! Added some explanation according to your suggestion [here](https://github.com/ChainAgnostic/namespaces/pull/117/commits/d128daedf783e31eef823ee20259c85fd77977f4), do you think it is more clear this way?